### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by michaelFreeman195** [akunzai/GSS.Authentication.CAS#233] [Q&A]
Hi,

 We are setting up SSO in .Net Core MVC. 

I've managed to get the middleware setup and can hit the cas server by adding the [authorize] attribute to a relevant action. The service url appears to be incorrect though when navigating to a lockdown page. 

The redirect results in this url (Note the signin-cas bit)
https://ourcasserver.com?service=ourtestsite.com/signin-cas?state=ABCDEFG

After successfully signing in to |CAS the redirect results in a 404 as "signin-cas" does not exist. 

I would expect the url to be
https://ourcasserver.com**?service=ourtestsite.com/lockeddownpage**

I have setup the service on the CAS server and also tried changing the protocol version via the appsettings in the .net core example app. 

Any assistance much appreciated. 

Thanks

**Comment by akunzai** [comment]
Can you create a minimal reproduce repo?

Let us can reproduce your issue with our CAS server.
